docs/_blog/why-bindings.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Why Do We Build Bindings?"
+description: "A post exploring why the Bitcoin Dev Kit Foundation produces language bindings for its libraries"
+authors:
+    - thunderbiscuit
+date: "2024-05-07"
+tags: ["bindings"]
+---
+
+This article explores the reasons why the Bitcoin Dev Kit Foundation supports a number of language bindings libraries as part of its core offering, and the challenges and decision tradeoffs we face along the way.
+
+We build language bindings for a number of use cases. One of the most common of those rests on a belief that as time goes on, applications of all kinds will need to interact with the bitcoin protocol. Many of those will be applications that are not "bitcoin-first" like wallets, but rather other kinds of applications that simply wish to integrate payments for their users (games, chat applications, content creation, etc.). These applications already have well-developed codebases and teams, seldom built entirely in Rust (BDK's first and core language). Our goal is to offer these teams and applications an all-inclusive dependency they can add to whatever technology stack they are using in production, and allow the integration of bitcoin-related capabilities without the need to completely change their tech stack or require the hire of full-time bitcoin experts.
```

Comment:
```suggestion
We build language bindings for a number of use cases. One of the most common of those rests on a belief that as time goes on, applications of all kinds will need to interact with the Bitcoin protocol. Many of those will be applications that are not "Bitcoin-first" like wallets, but rather other kinds of applications that simply wish to integrate payments for their users (games, chat applications, content creation, etc.). These applications already have well-developed codebases and teams, seldom built entirely in Rust (BDK's first and core language). Our goal is to offer these teams and applications an all-inclusive dependency they can add to whatever technology stack they are using in production, and allow the integration of Bitcoin-related capabilities without the need to completely change their tech stack or require the hire of full-time Bitcoin experts.
```

Bitcoin, title case, if you're talking about technology, protocol or phenomena.
bitcoin, lower case, if you're talking about amounts.

Also, here we could say a little about hyperbitcoinization.